content/attacks/posts/2023-05-29-Qubit.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2022-03-23
+title: Hackers Breach Qubit Finance Platform, Stealing $80 Million
+---
+
+## Summary
+
+On January 28, 2022, Qubit Finance, a project built on the BNB Chain (formerly known as Binance Smart Chain), announced a breach of its QBridge credit protocol on their Twitter page. The platform was hacked due to a specific vulnerability in the bridge. Hackers were able to mint an unlimited amount of xETH as collateral for loans on the Binance Smart Chain network.
+
+## Attackers
```

Comment:
add theories, description of their sophistication, anything that can help to narrow the suspect pool